drivers/gpio/gpio_pcal9535a.c

```diff
@@ -351,7 +351,7 @@ static int gpio_pcal9535a_config(struct device *dev, int access_op,
 {
 	int ret;
 
-#if (CONFIG_SYS_LOG_GPIO_PCAL9535A_LEVEL >= SYS_LOG_LEVEL_DEBUG)
+#if (CONFIG_LOG_GPIO_LEVEL >= LOG_LEVEL_DEBUG)
```

Comment:
you should be able to remove this #ifdef, compiler will cut it if logs are gone.